tests/rustdoc-ui/2024-doctests-checks.rs

```diff
@@ -0,0 +1,41 @@
+//@ check-pass
+//@ compile-flags: --test --test-args=--test-threads=1 -Zunstable-options --edition 2024
+//@ normalize-stdout-test: "tests/rustdoc-ui" -> "$$DIR"
+//@ normalize-stdout-test "finished in \d+\.\d+s" -> "finished in $$TIME"
+//@ normalize-stdout-test ".rs:\d+:\d+" -> ".rs:$$LINE:$$COL"
+
+/// This one should fail: crate attributes should remain crate attributes
+/// in standalone doctests.
+///
+/// ```compile_fail
+/// #![deny(missing_docs)]
```

Comment:
Do you mind removing this test? As I mentioned, it doesn't actually test anything, so it's a bit confusing to have it here. Or you could change it to test something else: Split it into two doctests: one for the deny and one for the struct Bar, so that it tests that the `missing_docs` doesn't affect other doctests.

Reply:
The deny is a crate attribute, so not really useful.

Reply:
Updated the doctest and just kept the part to ensure that the macro was not part of the merged doctests.